.github/workflows/gradle-run-unit-tests.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           merge-multiple: true
 
       - name: Upload code coverage to codecov
-        uses: codecov/codecov-action@v4.5.0
+        uses: codecov/codecov-action@ab904c41d6ece82784817410c45d8b8c02684457
```

Comment:
Last working version of codecov, rollback for now, since 4,x are broken https://github.com/codecov/codecov-action/issues/1463